ghauth.js

```diff
@@ -5,13 +5,19 @@ const read = promisify(require('read'))
 const hyperquest = require('hyperquest')
 const bl = require('bl')
 const appCfg = require('application-config')
+const querystring = require('querystring')
+const ora = require('ora')
+const logSymbols = require('log-symbols')
+const { Octokit } = require('@octokit/rest')
```

Comment:
Going to probably remove octokit.  Was having trouble making raw requests to the user endpoint and pulled this is a bandaid.